pkg/action/printer.go

```diff
@@ -45,20 +49,48 @@ func PrintRelease(out io.Writer, rel *release.Release) {
 			}
 			fmt.Fprintf(out, "TEST SUITE:     %s\n%s\n%s\n%s\n\n",
 				h.Name,
-				fmt.Sprintf("Last Started:   %s", h.LastRun.StartedAt),
-				fmt.Sprintf("Last Completed: %s", h.LastRun.CompletedAt),
+				fmt.Sprintf("Last Started:   %s", h.LastRun.StartedAt.Format(time.ANSIC)),
+				fmt.Sprintf("Last Completed: %s", h.LastRun.CompletedAt.Format(time.ANSIC)),
 				fmt.Sprintf("Phase:          %s", h.LastRun.Phase),
 			)
 		}
 	}
 
-	if strings.EqualFold(rel.Info.Description, "Dry run complete") {
+	if fullInfo {
```

Comment:
What kinds of information should be in `fullInfo` verses regular info.  If a value is added how would I make the decision?

Reply:
I _personally_ know what it is, but I think the best way around this is a comment. I'm thinking something like

> Anything printed in this block should be additional information about the release that is not necessarily needed for normal output (like the full manifest or all of the computed values). Basic information such as the name, revision, and release date should always be printed